src/content/charts.ts

```diff
@@ -0,0 +1,59 @@
+
+// TITLES
+// TITLES --> CASE GROWTH RATE
+export const CASE_GROWTH_RATE_TITLE = 'Infection growth rate';
+
+// TITLES --> HOSPITAL USAGE
+export const HOSPITAL_USAGE_TITLE = 'ICU headroom used';
+
+// TITLES --> POSITIVE TESTS
+export const POSITIVE_TESTS_TITLE = 'Positive test rate';
+
+
+// ZONE NAMES
+export const ZONE_LOW_NAME = 'Reduced';
+export const ZONE_MEDIUM_NAME = 'Moderate';
+export const ZONE_HIGH_NAME = 'Elevated';
+export const ZONE_UNKNOWN = 'Unknown';
+
+
+// LEGENDS
+// LEGENDS --> SUMMARY
+export const SUMMARY_LEGEND_LOW = 'Reduced risk based on reopening metrics.';
```

Comment:
Maybe HOMEPAGE_LEGEND_LOW ?  I don't inherently know what "summary legend" means.  And this too, doesn't really belong in a charts.ts file.